style.css

```diff
@@ -207,88 +183,123 @@ body{
 	margin: 100px auto;
 	position: relative;
 	background-color: white;
-	padding: 20px;
+	padding: 25px;
 	display: block;
 	max-width: 555px;
 	width: 100%;
-	height: 400px;
+	height: 100%;
 	border-radius: 5px;
 }
 
-/* custom style for cursor type on buttons*/
-input[type=submit]{
-	cursor: pointer;
+/* custom style for input boxes start*/
+input{
```

Comment:
#trialled custom styling for my input boxes so it would have an underline effect rather than a whole box